.gitignore

```diff
@@ -34,7 +34,3 @@ nosetests.xml
 .project
 .pydevproject
 *~
-odt/base/
```

Comment:
BTW, I added these to gitignore to avoid uploading the intermediate odts and pdfs.

Reply:
ignore specific files, like *.odt and so on